k6/run.sh

```
@@ -0,0 +1,35 @@
+#!/bin/bash
+# k6 테스트 실행 스크립트
+# 사용법: ./k6/run.sh <테스트파일명>
+# 예시:   ./k6/run.sh tests/my_test.js
+
+set -e
+
+# .env 파일 로드 (있으면)
+if [ -f "k6/.env" ]; then
+  export $(grep -v '^#' k6/.env | xargs)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)